_posts/2017-02-21-release-3-0-0.markdown

```diff
@@ -0,0 +1,170 @@
+
+Another thing about reports: atoum provides an advanced API to design your own reports. Reports in atoum use something 
+we call "fields" to display information. One of those field had its class named `void`. In latest PHP versions, `void` 
+is a reserved keyword thus we can't use it anymore in class names.
+
+**The `void` class has been renamed to `blank`**. If you wrote custom reports using this field, you will have to use 
+the new  class name.
+
+#### Assertions
+
+We changed some assertions and helpers in atoum and added a new one. 
+
+##### `exception` and `when`
+
+atoum `2.x` supported PHP `>= 5.3.3`. Because on lower version `$this` in closures was not bound to the current object 
+context, some assertions provided the test as an argument to closures.
+
+**This is not the case anymore: you will have to change your tests** if you used this feature.
+
+```php
+// atoum 2.x
+
+$this
+    ->when(function(atoum\test $test) { 
+        $test->testedInstance->doSomething();
+    })
+;
+
+$this
+    ->exception(function(atoum\test $test) { 
+        $test->testedInstance->doSomethingAndThrow();
+    })
+;
+
+// atoum 3.x
+
+$this
+    ->when(function() { 
+        $this->testedInstance->doSomething();
+    })
+; 
+
+$this
+    ->exception(function() { 
+        $this->testedInstance->doSomethingAndThrow();
+    })
+;
+```
+
+##### Asserting on generators
+
+Because generators are awesome, you should be able to use them and test them correctly. [@agallou][@agallou] did a great
+job on writing this new asserter.
+
+It will allow you to walk through any generator and assert on the yielded values. Here is an example:
+
+```php
+$this
+    ->generator($generator())
+        ->yields->array
+              ->isEqualTo(["1", "2", "3"])
+              ->contains("1")
+              ->string[0]->isEqualTo(1)
+        ->yields->integer
+              ->isZero
+        ->yields->variable->isNull
+        ->returns->variable->isEqualTo(42)
+        //...
+;
+```
+
+### Asides
+
+We also release the VIM plugin as a standalone component: it now has its [own repository][vim-plugin].
+
+We also released a Hombrew Tap with some formulae: [homebrew-atoum][homebrew-atoum].
+
+A new website is available: [http://extensions.atoum.org/](http://extensions.atoum.org/) where you will be able to 
+browse the list of extensions.
+
+### Extensions
+
+Some extensions have been updated:
+
+* The [config-extension][config-extension] has been released,
+* The [autoloop-extension][autoloop-extension] has been updated,
+* The [visibility-extension][visibility-extension] has been updated,
+* The [reports-extension][reports-extension] has been updated,
+* The [json-schema-extension][json-schema-extension] has been updated.
+
+We still need to update some extensions but the most used ones should already be OK.
+
+### Documentation
+
+We also updated missing parts of the documentation. Again, [Adrien Gallou][@agallou] and our two [doc masters][@doc] did an amazing job!
```

Comment:
one doc master, unfortunatelly. But the doc should be updated now ;) and we will need to create version inside it ;)